src/olympia/migrations/1138-db-schema-reviews.sql

```diff
@@ -0,0 +1,26 @@
+ALTER TABLE `reviews`
+    MODIFY `created` datetime(6) NOT NULL,
+    MODIFY `modified` datetime(6) NOT NULL,
+    MODIFY `id` int(10) unsigned NOT NULL,
```

Comment:
I got `Cannot change column 'id': used in a foreign key constraint 'akismet_reports_rating_instance_id_d282058c_fk_reviews_id' of table 'olympia_dev_copy.akismet_reports` when running this

Reply:
https://github.com/mozilla/addons-server/pull/12272/commits/a5ae7521db48294a7852c3bb971e907109fe4683